packages/common/styles/_mixins.scss

```diff
@@ -0,0 +1,49 @@
+@import './tokens';
+
+// Font fallbacks
+// Do not use outside this file - these are temporary.
+// TODO: Remove these fallbacks and their usages once fonts
```

Comment:
Why would we remove the fallbacks after we're hosting the fonts? If we were using the design system outside of somewhere that could load the fonts, it would be nice if it used some reasonable default

Reply:
that's a good point. i assumed we'd want to remove them because it's a "hard" dependency on code studio, but i suppose fonts aren't really a hard dependency since they fallback gracefully so there's no harm in leaving them